src/components/analytics/metrics-table.vue

```diff
@@ -56,6 +56,10 @@ export default {
   .metric-value {
     text-align: right;
   }
+
+  .total-col {
+    width: 50px;
```

Comment:
Flexbox doesn't totally seem to respect this width, but making it this small does seem to get it to choose a consistent width.